### DISCUSSION TITLE
如何部署到heroku上？

### DISCUSSION THREAD

**Discussion by longdanie** [AlistGo/alist#475] [Q&A]
请问如何部署到heroku上？ 前面帖子里提到的 sbwml 的仓库里安装不上。

**Comment by xhofe** [comment]
先fork https://github.com/alist-org/alist-heroku
然后从自己仓库deploy。

**Reply by longdanie** [reply]
多谢指导，我已经搞掂了。可惜这个方法不能及时更新你的最新代码。
